newsfragments/201.feature.rst

```diff
@@ -0,0 +1,4 @@
+Added a new variant of apply_formatter_if to eth_utils curried module.
+Also added text_if_str and apply_formatters_to_dict.
+
+Updated mypy to 0.720, and added a deprecation warning check
```

Comment:
This could be split up into two different fragments, so like this line could go in 201.internal.rst.

Reply:
Ah, smart!